src/styles/Modal.css

```diff
@@ -37,3 +37,41 @@
 .cancelDelete:hover {
   background-color: #ccc;
 }
+
+.detailsModal {
+  margin-bottom: 20px;
+  padding: 30px 20px;
+  background-color: #98c1d9;
+  border: 1px solid #293241;
+  border-radius: 6px;
+}
+.detailsModal h1 {
+  margin-top: 0;
+}
+.detailsModal h3 {
+  /* display: inline-block;
```

Comment:
This may be temporary, but let's remove commented out code if we decide not to move forward with it to keep things nice and clean.